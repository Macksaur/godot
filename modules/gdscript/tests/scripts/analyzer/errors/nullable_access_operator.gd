class A:
    var b: A?
func test():
    var foo := A.new()
    foo?.b.b
