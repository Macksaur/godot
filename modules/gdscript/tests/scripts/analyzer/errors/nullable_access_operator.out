GDTEST_ANALYZER_ERROR
Potentially unsafe to access nullable "foo?.b" with attribute "b".
